exposee-app/models/users.js

```diff
@@ -14,5 +14,8 @@ export const User = sequelize.define('User', {
   password: {
     type: DataTypes.STRING,
     allowNull: false
+  },
+  profilePicture:{
```

Comment:
we should stay consistent within a codebase, typically we've designed the columns to be snake case so we should stick with that; profile_picture instead of profilePicture

eg.
```
 created_at: {
    type: DataTypes.TIME,
    allowNull: false 
  },
  is_live: {
    type: DataTypes.BOOLEAN,
    allowNull: false,
  },
  is_saved:{
  ```